README.md

```diff
@@ -26,9 +26,6 @@ To integrate using [CocoaPods](https://cocoapods.org), add the following line to
 ```ruby
 pod 'PopupBridge'
 ```
-### Carthage
```

Comment:
None of our release or build steps build or release frameworks for Carthage. Seems like we don't actually support Carthage unless I am missing something?

Reply:
You're right we don't verify anywhere that `carthage update` succeeds, but it very well could be used with the SDK still.

Reply:
Wouldn't we still need to distribute the framework for Carthage? For example, when we run `zip -r Braintree.xcframework.zip Carthage` in our BT release, once that's done we upload that zip to the release it shows up here: https://github.com/braintree/braintree_ios/releases/tag/5.22.0 as a downloadable zip. With popup bridge, we don't do these steps and there is no zip of the framework here: https://github.com/braintree/popup-bridge-ios/releases/tag/1.2.0

Again, maybe I've totally misunderstood Carthage magic this while time but my understanding was that we'd need that zip of the framework to exist in order for a merchant to use the SDK with Carthage.

Reply:
Ah good question - so Carthage doesn't require those `zip` files be attached to the releases. Those are a nice-to-have feature that speeds up merchants time pulling down the SDK via Carthage. We used to not ship them with our releases for awhile, but a merchant requested we add it awhile ago (https://github.com/braintree/braintree_ios/issues/543). 

(Also - If a Carthage user wants to bypass using our pre-built binaries, they can pass the flag `carthage update --no-use-binaries`)

Reply:
So since this SDK has no dependencies, it doesn't need an SDK-level Cartfile. A merchant could still be including this SDK in their app via Carthage by specifying our URL in their Cartfile and building from source (like: `github "braintree/popup-bridge-ios" ~> 5.5`).

Reply:
🤯 learn something new about package managers every day! Thanks for the explanation as always, this whole time I assumed that zipping the binaries was required to use Carthage. Doesn't seem like we've gotten any inbounds about it on this SDK, assuming because it's so small, but we could always add it here in the future to have consistencies across all the SDKs.

Reply:
Reverted this here: https://github.com/braintree/popup-bridge-ios/pull/57/commits/f7c255efbfbc3de1de2543e00182b1894204610d